src/vdc/Vdc.cc

```diff
@@ -397,34 +397,61 @@ int Vdc::del_group(int group_id, string& error_msg)
 /* ------------------------------------------------------------------------ */
 /* ------------------------------------------------------------------------ */
 
+void ResourceSet::insert_defult_rules(string default_vdc_acl, PoolObjectSQL::ObjectType type)
+{
+    vector<string> vdc_acl;
+    vector<string>::const_iterator it;
+    long long op = AuthRequest::NONE;
+
+    vdc_acl = one_util::split(default_vdc_acl, '+', true);
+
+    for (it = vdc_acl.begin(); it != vdc_acl.end(); ++it)
+    {
+        if ( *it != "" && *it != "-" && AuthRequest::str_to_operation(*it) != AuthRequest::NONE )
```

Comment:
AuthRequest::str_to_operation(*it)  is called twice maybe it'd be better to saved in a variable:

AuthRequest::Operation user_op =  AuthRequest::str_to_operation(*it);

and then use that